polygeist/tools/cgeist/Options.h

```diff
@@ -202,4 +202,8 @@ static llvm::cl::opt<std::string> McpuOpt("mcpu", llvm::cl::init(""),
                                           llvm::cl::desc("Target CPU"),
                                           llvm::cl::cat(ToolOptions));
 
+llvm::cl::opt<bool> AddOptionalMangledFunctionName(
+    "fadd-mangled-function-name", llvm::cl::init(false),
```

Comment:
We need to test the old functionality so all lit tests should also when this flag is used. 
Actually, I propose we default this flag to true in this PR and add RUN lines to all affected lit tests to test the new infrastructure. 
When we are happy with the new infrastructure we might flip the default, and subsequently remove the flag alltogheter.

Reply:
I have no issues with that. I'll do. Thanks for the suggestion.